tests/tool_test.py

```diff
@@ -107,21 +107,31 @@ def patchConfigurators(mockConfigurers):
 
 class PasswdConfiguratorTest(VdsmTestCase):
     def testCheckIsConfiguredNo(self):
-        tmpfile = tempfile.mktemp()
-        with open(tmpfile, 'w') as f:
+        tmpfile = tempfile.NamedTemporaryFile(mode='w+')
+        with tmpfile as f:
             f.write("\n"
                     "\n"
                     "mech_list: gssapi\n")
+            f.flush()
 
-        passwd._SASL2_CONF = tmpfile
-        self.assertEqual(passwd.libvirt_sasl_isconfigured(), NO)
+            passwd._SASL2_CONF = tmpfile.name
```

Comment:
I know that the original test used the same constructs, but this is actually horrible - we modify the `passwd._SASL2_CONF` for the _whole lifetime of the test module_. These kind of changes should be scoped - you change the attribute before the tests starts and then you set it back to the original value once the single test is completed.
This PR is rather urgent so I wouldn't block on it, but we should improve this test module in subsequent PRs.

Reply:
will be handle in another PR